jest.config.ts

```diff
@@ -0,0 +1,22 @@
+const getProject = (type: string, color: string) => ({
+  displayName: {
+    name: type.toUpperCase(),
+    color,
+  },
+  moduleNameMapper: {
+    '@/(.*)$': '<rootDir>/$1',
+  },
+  preset: 'ts-jest',
+  setupFiles: ['<rootDir>/tests/setup.ts'],
+  testMatch: [`<rootDir>/tests/${type}/**/*.spec.ts`],
+  testEnvironment: 'node',
+});
+
+const jestConfig = {
```

Comment:
Jest provides a `Config` type that we should use here. See https://jestjs.io/docs/configuration